.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -145,14 +141,6 @@ Release blog is ready | :red_circle: |   |
 - [ ] Advertise on Social Media.
 - [ ]  Post on public slack and Github Release issue.
 
-### Post-Release activities
```

Comment:
We still need these right @gaiksaya ?
For Release Tags and Input Manifest Update we have an automated workflow which we can add here. Also for Helm and Ansible Playbook release. today we do them after the release.

Reply:
It was a duplicate places in checklist. See https://github.com/opensearch-project/opensearch-build/pull/5031/files#diff-38e15f3a510b3e0a8b537c78468a4c2db29c2f00c55abbd0a310fbcfe9c395c2R148